app/models/item_order.rb

```diff
@@ -32,7 +33,9 @@ def unfulfilled?
   end
 
   def items
-    ItemOrder.where('order_id = ?', order.id)
+    binding.pry
+    order.order_items
+    # ItemOrder.where('order_id = ?', order.id)
```

Comment:
You still have a pry and commented out line here

Reply:
good catch

Reply:
excellent, i'll get that removed